src/main/steps/cookie-banner/template.njk

```diff
@@ -1,51 +1,67 @@
 {% from "govuk/components/cookie-banner/macro.njk" import govukCookieBanner %}
 
 {% set cookieMessageHTML %}
-  <p>{{ cookiesLine1 }}</p>
-  <p>{{ cookiesLine2 }}</p>
+  <p class="govuk-body">{{ cookiesLine1 }}</p>
+  <p class="govuk-body">{{ cookiesLine2 }}</p>
 {% endset %}
 
-<div class="global-cookie-message" id="cm-cookie-banner" >
-  {{ govukCookieBanner({
-    ariaLabel: cookiesHeading + " " + serviceName,
-    messages: [
+    {% set acceptHtml %}
+  <p class="govuk-body">You’ve accepted additional cookies. You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>
```

Comment:
what about welsh for this?

Reply:
No Alok we dont have welsh for this. But I have given welsh infrastructure for this as of now.